src/legacy/core_plugins/console/public/src/sense_editor_resize.js

```diff
@@ -17,10 +17,23 @@
  * under the License.
  */
 
-import { ResizeChecker } from 'ui/resize_checker';
+import $ from 'jquery';
+import { ResizeChecker } from '../../../../../plugins/kibana_utils/public';
 
 export function applyResizeCheckerToEditors($scope, $el, ...editors) {
-  const checker = new ResizeChecker($el);
+  // Only this function uses ResizeChecker with jQuery.
+  // So, we're extracting element from jQuery object.
+  const checker = new ResizeChecker(extractElement($el));
   checker.on('resize', () => editors.forEach(e => e.resize()));
   $scope.$on('$destroy', () => checker.destroy());
 }
+
+function extractElement(el) {
+  const $el = $(el);
```

Comment:
This line is unneeded, as the incoming $el is already a jquery object.
This also makes the 'jquery' import unnecessary.

Reply:
Thanks. I'll fix that soon.